web-test-runner.config.js

```diff
@@ -1,38 +1,13 @@
-import { playwrightLauncher } from '@web/test-runner-playwright';
+import { createConfig, getBrowsers } from '@brightspace-ui/testing';
```

Comment:
May want to hold off on merging until my changes go in... at which point this will be:

```suggestion
import { createConfig, getBrowsers } from '@brightspace-ui/testing/wtr-config.js';
```